### PR TITLE
Increased unit test coverage in page-index.spec.js

### PR DESCRIPTION
**Pull request by ColinBeeby-Developer**
## Description
Additional tests have been added to increase the code coverage stats for this product

### Related issue
No related issue. Just increasing the code coverage stats.

## Checklist

- [ X ] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ X ] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [ ] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [ X ] CHANGELOG entry
- [ ] [Whats new page](https://service-manual.nhs.uk/whats-new) entry
- [ ] Page updated date

### CODE REVIEW

File: tests/page-index.spec.js [hunk 1]
```diff
@@ -26,15 +61,180 @@ test('test get index', () => {
...
+    h3: 'for building user interfaces',
+    extra: 'Some new extra text',
+  }, {
+    title: 'Lodash',
+    h2: 'A modern JavaScript utility library',
+    h3: 'delivering modularity, performance & extras',
+    extra: 'Some new extra text',
+  }];
+
+  const idx = lunr(function () {
```
Comment (will-dyson-nhsd) [inline tests/page-index.spec.js:155]: ESLint warning: https://eslint.org/docs/latest/rules/func-names

File: tests/page-index.spec.js [hunk 2]
```diff
@@ -26,15 +61,180 @@ test('test get index', () => {
...
+    this.ref('name');
+    this.field('text');
+
+    documents.forEach(function (doc) {
```
Comment (will-dyson-nhsd) [inline tests/page-index.spec.js:159]: ESLint warning: https://eslint.org/docs/latest/rules/func-names

File: tests/page-index.spec.js (hunk 1 above)
Comment (ColinBeeby-Developer) [reply tests/page-index.spec.js]: Function name has now been added

File: tests/page-index.spec.js (hunk 2 above)
Comment (ColinBeeby-Developer) [reply tests/page-index.spec.js]: Function name has now been added

**Review by will-dyson-nhsd — APPROVED** [review]

**Pull request merged**